components/DiscordBar.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react'
+import styles from '../scss/discordBar.module.scss'
+type Member = {
+  id: string
+  username: string
+  status: string
+  avatar_url: string
+}
+interface discordJSON {
+  presence_count: number
+  members: Member[]
+}
+const ONE_MINUTE = 60 * 1000
+const Member: React.FC<{ avatar: string }> = ({ avatar }) => {
+  return (
+    <div className="d-inline-block">
+      <img src={avatar} className={`${styles['member-avatar']} mx-1`} />
+    </div>
+  )
+}
+const DiscordBar: React.FC = () => {
+  const [data, setData] = useState<discordJSON>({
+    presence_count: 1,
+    members: []
+  })
+  const getData = async () => {
+    try {
+      let data = await fetch(
+        'https://discord.com/api/guilds/828783458469675019/widget.json'
+      )
+      data = await data.json()
+      setData((data as unknown) as discordJSON)
+      setTimeout(getData, ONE_MINUTE)
+    } catch (e) {
+      setTimeout(getData, ONE_MINUTE)
+    }
+  }
+  useEffect(() => {
+    getData()
+  }, [])
+  const reshuffle = (data: discordJSON) => {
```

Comment:
This reshuffles members list, slices the first 5 and renders it, so you will see different users in different order every time. Sorting isn't very expensive with our members count and it looks nicer this way.

Reply:
would have been better if this was a comment in your code... because when people read the code later, they may see your comment you left in this PR.

If you have to explain yourself for code reviewers, this explanation should be a comment in the code :)

Reply:
I added comment in this case but it's hard to strike balance between 'it should be obvious from the code' and 'too much comments'.